### ISSUE TITLE
How do I generate string sequence?

### ISSUE THREAD

**Issue by OpusK** [eProsima/Micro-XRCE-DDS-Gen#4]
Hi,

For example, here is the sensor_msgs/JointState topic:

```
struct JointState
{
//  std_msgs::msg::dds_::Header_ header;
  sequence<string> name;
  sequence<double> position;
  sequence<double> velocity;
  sequence<double> effort;
};  // struct JointState
```
When I generate this IDL file, I get the following code, but the problem is in sequence <string>.

```c
bool JointState_serialize_topic(MicroBuffer* writer, const JointState* topic)
{
    (void) serialize_sequence_char(writer, topic->name_data, topic->name_size);
    ... 
}

bool JointState_deserialize_topic(MicroBuffer* reader, JointState* topic)
{
   (void) deserialize_sequence_char(writer, topic->name_data, &topic->name_size);
   ...
}

uint32_t JointState_size_of_topic(const JointState* topic, uint32_t size)
{
    size += 4 + get_alignment(size, 4);
    for(size_t a = 0; a < topic->name.size(); ++a)
    {
        size += 4 + eprosima::fastcdr::Cdr::alignment(size, 4) + topic->name.at(a).size() + 1;
    }
    ...
}
```

The sequence_char can not represent a string sequence once.
Also, in size, a function of fastcdr is called, which can not be found in microRTPS.

It was recently tested with 1.0.0beta2.
Is my method wrong?

**Comment by richiware** [comment]
You are doing well. Short time ago we made a change about strings in micrortpsgen and it should break sequence of 
string.  Thanks for the reporting. We will work in solving it.
↪ 2018-08-01 mié 09:23, Kei <notifications@github.com>:

**Comment by OpusK** [comment]
Hi, @richiware.
Is there any way to handle String sequences in this situation?

**Comment by lemunozm** [comment]
Hi @OpusK,

The string sequences are now available in the pull request #6
